pkg/test/framework/suite.go

```diff
@@ -177,6 +182,11 @@ func (s *Suite) run() (errLevel int) {
 
 	ctx := rt.suiteContext()
 
+	// Skip the test if its explicitly skipped
+	if s.skip != "" {
+		scopes.Framework.Infof("Skipping suite %q: %s", ctx.Settings().TestID, s.skip)
```

Comment:
Is this the only place s.skip is used? How will the test get skipped?

Reply:
Big mistake here, missing a `return 0`. Thanks for catching this!